pandas/tests/io/xml/test_xml.py

```diff
@@ -1102,6 +1102,10 @@ def test_unsuported_compression(datapath, parser):
 @tm.network
 @td.skip_if_no("s3fs")
 @td.skip_if_no("lxml")
+@pytest.mark.skipif(
+    os.environ.get("PANDAS_CI", "0") == "1",
+    reason="2022.1.17: Hanging on the CI min versions build.",
+)
```

Comment:
You’re confident this is the culprit?

Reply:
Fairly. I'll run one more time removing the `-v`

Reply:
Sounds good to me.